src/main/java/org/kpmp/geneExpressionSummary/ParticipantId.java

```diff
@@ -0,0 +1,29 @@
+package org.kpmp.geneExpressionSummary;
```

Comment:
Even though this is a POJO (Plain Old Java Object) we like to have tests around the methods.